TeamCode/src/main/java/org/firstinspires/ftc/teamcode/constants/DrivetrainConstants.java

```diff
@@ -4,4 +4,6 @@ public class DrivetrainConstants {
     public static final String MOTOR_RIGHT = "Rmotor";
     public static final String MOTOR_LEFT = "Lmotor";
     public static final String IMU = "imu";
+    public static final double MOTORS_REDUCTION = (float) 1/4 * 1/5;
```

Comment:
minor:
in order to separate the constants without creating a java file for each, add a subsystem constant class inside of the global constants. for instance:
public final class GlobalConstants {

  public static final class Drivetrain {
     // add here drivetrain constants
   }
}

Reply:
We think that is better with different files. Because if we have many Constants classes in the same file, it would be difficult to find the classes that we want. So we will not change it.